partiql-types/Cargo.toml

```diff
@@ -32,6 +32,7 @@ thiserror = "1.*"
 indexmap = "2.2"
 
 derivative = "2.2"
+lazy_static = "1.5.0"
```

Comment:
lazy_static usage is currently explicitly disallowed, as `OnceCell` is preferred: https://github.com/partiql/partiql-lang-rust/blob/684fff3092311722c15b5b86506c375a49f78d1a/deny.toml#L126-L128

Reply:
Yeah this one pulled in by accident, thanks for the fix!